wrappers/services.go

```diff
@@ -335,7 +335,27 @@ func StopServices(apps []*snap.AppInfo, reason snap.ServiceStopReason, inter int
 	}
 
 	return nil
+}
+
+// CurrentSnapServiceStates builds a map of service names and whether the snap
+// is enabled or disabled
+func CurrentSnapServiceStates(s *snap.Info, inter interacter) (map[string]bool, error) {
```

Comment:
I think you can drop the `Snap` part. How about `CurrentServicesEnableState` or just `ServicesEnableState`?

Reply:
I like `ServicesEnableState`, I'll go with that I think